iris_grib/tests/integration/load_convert/test_data_section.py

```diff
@@ -68,5 +69,16 @@ def test_grid_complex_spatial_differencing(self):
         self.assertCMLApproxData(cube)
 
 
+class TestDataProxy(tests.IrisGribTest):
+    def test_data_representation__no_bitsPerValue(self):
+        """Ensures that zero data is auto-generated."""
+        path = tests.get_data_path(
```

Comment:
Can we just add a comment, that this test file contains an empty data section ?

Reply:
@pp-mo See https://github.com/SciTools/iris-grib/pull/362/commits/92435d865722e54382c7d530b189bc6a46d0b703

Reply:
Wow, that's comprehensive !